src/Common/src/CoreLib/System/Globalization/DateTimeParse.cs

```diff
@@ -1160,7 +1169,21 @@ private static bool Lex(DS dps, ref __DTString str, ref DateTimeToken dtok, ref
             return true;
         }
 
-        private static bool VerifyValidPunctuation(ref __DTString str)
+        private static Calendar GetJapaneseCalendarDefaultInstance()
+        {
+            if (GlobalizationMode.Invariant)
+                throw new PlatformNotSupportedException();
+            return JapaneseCalendar.GetDefaultInstance();
+        }
+
```

Comment:
There are more references to these calendars in `CultureInfo.GetCalendarInstanceRare`. Do those need to be changed as well?